src/test/java/com/kmorarash/hwJDBC/UserServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package com.kmorarash.hwJDBC;
+
+import com.kmorarash.hwJDBC.dto.UserRegistrationDto;
+import com.kmorarash.hwJDBC.dto.UserResponseDto;
+import com.kmorarash.hwJDBC.model.User;
+import com.kmorarash.hwJDBC.service.UserService;
+import com.kmorarash.hwJDBC.service.UserServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import com.kmorarash.hwJDBC.repository.UserRepository;
+import com.kmorarash.hwJDBC.validator.UserValidator;
+
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
```

Comment:
It is better to avoid using star import. Please see message in the TG chat where I shared an example of how IntelliJ Idea can be configured to avoid using star import